HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2022,7 +2035,18 @@ def self.process_heat_pump_adjustment(hvac_sizing_values, weather, hvac_heating,
       heating_db = @hpxml.header.manualj_heating_design_temp
     end
 
-    heat_cap_rated = (heating_load / MathTools.biquadratic(@heat_setpoint, heating_db, coefficients)) / capacity_ratio
+    if hvac_heating.compressor_type == HPXML::HVACCompressorTypeVariableSpeed
+      capacity_retention_temp_1speed, capacity_retention_fraction_1speed = HVAC.get_default_heating_capacity_retention(HPXML::HVACCompressorTypeSingleStage)
+      coefficients_1speed = HVAC.get_heat_cap_eir_ft_spec(HPXML::HVACCompressorTypeSingleStage, capacity_retention_temp_1speed, capacity_retention_fraction_1speed)[0][0]
+      heating_capacity_retention_temperature, heating_capacity_retention_fraction = HVAC.get_default_heating_capacity_retention(hvac_heating.compressor_type, hvac_heating.heating_efficiency_hspf)
```

Comment:
This doesn't look right. Why are we always using the _default_ capacity retention? If the HPXML file has a user-entered capacity retention factor, shouldn't we use it?

Reply:
Perhaps this could be (untested code)....

```suggestion
      if hvac_heating.heating_capacity_retention_fraction.nil?
        heating_capacity_retention_temperature, heating_capacity_retention_fraction = HVAC.get_default_heating_capacity_retention(hvac_heating.compressor_type, hvac_heating.heating_efficiency_hspf)
      else
        heating_capacity_retention_temperature, heating_capacity_retention_fraction = hvac_heating.heating_capacity_retention_temp, hvac_heating.heating_capacity_retention_fraction 
      end
```

Reply:
Yes that's here mostly because I used to set retention fraction and temperature to be nil when detailed performance data is provided, and the 17F capacity can also be entered instead of retention fractions. I think there's a better way to handle that now.

Reply:
Also, I'm seeing pretty similar equations (adjustment for indoor conditions) three times in hvac_sizing.rb. It should be consolidated into a method.